SumoLogic.Logging.NLog/SumoLogic.Logging.NLog.csproj

```diff
@@ -30,6 +30,9 @@
  - Update copyrights
  - New model for nuget metadata</PackageReleaseNotes>
   </PropertyGroup>
+  <PropertyGroup>
+    <VersionSuffix>beta1</VersionSuffix>

```

Comment:
Personally, I'd prefer this to be in Sumologic.Logging.CI.ps1 since all version definitions are done there.

Reply:
Without packaging everything separately, I couldn't figure out a way to make the dependencies pick up the version without the suffix using just the ps1. If you're aware of a workaround, let me know and I can update the PR!

Reply:
Right, it has to be in the json file that dotnet restore creates, which means this really has to live in the csproj. Alrighty, makes sense.